src/scripts/main.js

```diff
@@ -1,5 +1,34 @@
 'use strict';
 
+const wall = document.querySelector('.wall');
+const spider = document.querySelector('.spider');
+
 document.addEventListener('click', (e) => {
-  // write code here
+  const wallCoords = wall.getBoundingClientRect();
+  const spiderCoords = spider.getBoundingClientRect();
+  const border = parseInt(
+    window.getComputedStyle(wall).borderWidth.split('px').join(''),
+  );
+  let spiderY =
+    e.clientY - wallCoords.top - parseInt(spider.offsetWidth) / 2 - border;
+  let spiderX =
+    e.clientX - wallCoords.left - parseInt(spider.offsetWidth) / 2 - border;
```

Comment:
Similarly, the calculation of `spiderX` uses `spider.offsetWidth`, which is correct for horizontal positioning. Ensure that this is the intended behavior.